components/retro-go/rg_network.c

```diff
@@ -277,8 +283,7 @@ bool rg_network_init(void)
     // Init event loop first
     esp_err_t err;
     TRY(esp_event_loop_create_default());
-    TRY(esp_event_handler_register(WIFI_EVENT, ESP_EVENT_ANY_ID, &network_event_handler, NULL));
-    TRY(esp_event_handler_register(IP_EVENT, IP_EVENT_STA_GOT_IP, &network_event_handler, NULL));
+    TRY(esp_event_handler_register(ESP_EVENT_ANY_BASE, ESP_EVENT_ANY_ID, &network_event_handler, NULL));
```

Comment:
Switching to `ESP_EVENT_ANY_ID` is a good idea but are you sure it's necessary to switch to `ESP_EVENT_ANY_BASE`?

The event loop is used for more than networking so listening only for what we need seems better. Is it because esp-idf removed the constants `WIFI_EVENT` and `IP_EVENT`?

Reply:
Thanks that seems perfect.